src/ui/common/src/components/workflows/ReactFlowCanvas.tsx

```diff
@@ -42,7 +42,7 @@ const ReactFlowCanvas: React.FC<ReactFlowCanvasProps> = ({
     setTimeout(fitView, 200);
   }, [openSideSheetState]);
 
-  useEffect(fitView);
+  useEffect(fitView, []);
```

Comment:
Should be resolved in https://github.com/aqueducthq/aqueduct/pull/23 can you update here to match?